diesel_compile_tests/tests/ui/belongs_to_second_parent.stderr

```diff
@@ -0,0 +1,19 @@
+warning: belongs_to takes a single parent. Change
+    belongs_to(Parent1, Parent2)
+to
+    belongs_to(Parent1)
+    belongs_to(Parent2)
+  --> $DIR/belongs_to_second_parent.rs:23:19
+   |
+23 | #[belongs_to(Bar, Baz)]
+   |                   ^^^
+
+error[E0433]: failed to resolve: use of undeclared type or module `foos`
```

Comment:
This error message does not belong here. Seems like the table should be named `foos` not `foo`.

Reply:
This was taken from the existing `belongs_to_invalid_option_syntax` test. I've fixed it there as well.